perf/_runner.py

```diff
@@ -666,6 +676,36 @@ def _spawn_worker(self, calibrate=False):
 
         return _load_suite_from_pipe(bench_json)
 
+    def _spawn_worker_windows(self, calibrate=False):
+        import tempfile
+        wpipe, tmp_file = tempfile.mkstemp()
+        os.close(wpipe)
+        os.remove(tmp_file)  # remove for subprocess to create
+
+        cmd = self._worker_cmd(calibrate, None, output=tmp_file)
+        env = create_environ(self.args.inherit_environ,
+                             self.args.locale)
+
+        try:
+            proc = subprocess.Popen(cmd, env=env)
+
+            with popen_killer(proc):
+                exitcode = proc.wait()
+
+            if exitcode:
+                raise RuntimeError("%s failed with exit code %s"
+                                   % (cmd[0], exitcode))
+
+            with open(tmp_file, "r", encoding="utf8") as rfile:
```

Comment:
On Python 2, you should avoid the encoding parameter, and read bytes: see existing _spawn_worker() function.